docs/guides-publish.md

```diff
@@ -166,25 +162,12 @@ Sample output:
  ✔ 2.0.3: 0xb4fa71b3352D48AA93D34d085f87bb4aF0cE6Ab5 ipfs:QmcgUz9PXaZwvA3m7fXPgjsEVKteuivLNSCDvxKGv8ztMa
 ```
 
-## More about environments
-
-The default environments used by aragonCLI are defined in [`environments.default.json`](https://github.com/aragon/aragon-cli/blob/master/packages/aragon-cli/config/environments.default.json). This file has 4 environments:
-
-| Environment    | Network   |
-| -------------- | --------- |
-| aragon:local   | localhost |
-| aragon:rinkeby | rinkeby   |
-| aragon:staging | rinkeby   |
-| aragon:mainnet | mainnet   |
-
-In case you have an `arapp.json` file in your app directory, aragonCLI will use your local configuration over the default.
-
 ## Building frontends
 
 Your application's frontend will have another build script associated with it, to transpile, bundle, and pack all of its assets (e.g. scripts, images, fonts, etc) together.
 
-If you've used one of the boilerplates, it's likely this has already been set up for you with [`parcel-bundler`](https://parceljs.org) and [aragonUI](/docs/aragonui-intro).
+If you've used the Aragon react boilerplate, this has already been set up for you with [`parcel-bundler`](https://parceljs.org) and [aragonUI](/docs/aragonui-intro).
 
 If you need to add, modify, or remove assets or the way the frontend is built, it's important to remember to **always** use **relative paths** to serve those assets. Usually, this can be accomplished by adding a `./` in front of the path.
 
-This is important because the Aragon client usually fetches all of an app's assets via an IPFS gateway, and non-relative paths break gateway resolutions. You can test this for yourself by attempting to access your app when it's published by going to an IPFS gateway (see [tips and tricks of propagating IPFS content](/docs/guides-faq#propagating-your-content-hash-through-ipfs)) and making sure its assets are being loaded correctly.
+This is important because in production, the Aragon client usually fetches all of an app's assets via an IPFS gateway, and non-relative paths break gateway resolutions. You can test this for yourself by attempting to access your app when it's published by going to an IPFS gateway and making sure its assets are being loaded correctly.
```

Comment:
Do we still have a relevant link for more information on how to propagate IPFS content?

We can add information about e.g. pinata as a pinning service :).

Reply:
> Do we still have a relevant link for more information on how to propagate IPFS content?

I could put the link back, we still have the information in the troubleshooting section. However, is this something that we want to recommend to users? 


> We can add information about e.g. pinata as a pinning service :).

Good idea! I added them to the “Setup” section.

Reply:
I don't think propagation is needed here, but we should add some more info in the troubleshooting section with those same links :).